### PR TITLE
refactor(breadcrumbs): implement Chakra's `Breadcrumb` component

### LINKED ISSUE #8772: Implement Chakra's Breadcrumbs component
### Is your feature request related to a problem? Please describe.

Currently we have [a custom implementation](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/Breadcrumbs.tsx) of the Breadcrumbs component.

### Describe the solution you'd like

Replace the current implementation using the [Chakra's Breadcrumbs component](https://chakra-ui.com/docs/components/breadcrumb/usage).

### Describe alternatives you've considered

None.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description
Implements Chakra's `Breadcrumb` multi-component in the custom `Breadcrumbs` component.

## Related Issue
Closes #8772

### CODE REVIEW

File: src/components/Breadcrumbs.tsx [hunk 1]
```diff
@@ -43,56 +48,49 @@ const Breadcrumbs: React.FC<IProps> = ({
       : ""
 
     return {
-      fullPath: slugChunk.slice(0, idx + 2 + startDepth).join("/") + "/",
```
Comment (pettinarip) [inline src/components/Breadcrumbs.tsx:46]: I think we need this final `/`

It is not showing as active [here](https://ethereumorgwebsitedev01-tylerapfleddererethereumo29374.gatsbyjs.io/en/contributing/translation-program/acknowledgements/):
image

**Review by pettinarip — CHANGES_REQUESTED** [review]
Looks good but I found a small issue with the active styles.

File: src/components/Breadcrumbs.tsx (hunk 1 above)
Comment (TylerAPfledderer) [reply src/components/Breadcrumbs.tsx:46]: Whoops, Got it!

**Review by pettinarip — APPROVED** [review]
yeyy! 🎉

**Pull request merged**